gridstatus/ieso.py

```diff
@@ -0,0 +1,466 @@
+import time
+import xml.etree.ElementTree as ET
+
+import pandas as pd
+import requests
+
+from gridstatus import utils
+from gridstatus.base import ISOBase, NotSupported
+from gridstatus.decorators import support_date_range
+from gridstatus.gs_logging import log
+
+"""LOAD CONSTANTS"""
+# Load hourly files go back 30 days
+MAXIMUM_DAYS_IN_PAST_FOR_LOAD = 30
+LOAD_INDEX_URL = "http://reports.ieso.ca/public/RealtimeConstTotals"
+
+# Each load file covers one hour. We have to use the xml instead of the csv because
+# the csv does not have demand for Ontario.
+LOAD_TEMPLATE_URL = f"{LOAD_INDEX_URL}/PUB_RealtimeConstTotals_YYYYMMDDHH.xml"
+
+
+"""LOAD FORECAST CONSTANTS"""
+# There's only one load forecast for Ontario. This data goes back 5 days in the past
+# and only covers the current date.
```

Comment:
this multi-day endpoint cover the current market day and the next day, right?  
<img width="658" alt="image" src="https://github.com/kmax12/gridstatus/assets/371792/d1886df3-a07f-4d53-b8e7-559d24f23672">

Reply:
Yeah, sorry this comment was before I realized the forecast went through tomorrow.